ths.go

```diff
@@ -0,0 +1,36 @@
+package nmea
+
+const (
+	// TypeTHS type for THS sentences
+	TypeTHS = "THS"
+	// AutonomousTHS autonomous ths heading
+	AutonomousTHS = "A"
+	// EstimatedTHS estimated (dead reckoning) THS heading
+	EstimatedTHS = "E"
+	// ManualTHS manual input THS heading
+	ManualTHS = "M"
+	// SimulatorTHS simulated THS heading
+	SimulatorTHS = "S"
+	// NotValidTHS not valid THS heading (or standby)
+	NotValidTHS = "V"
```

Comment:
I think this should be `InvalidTHS`  to keep consistent with the other messages.

Reply:
Done. Consistency is a good thing.